Tests/WSSRV004.ps1

```diff
@@ -0,0 +1,84 @@
+#requires -Modules ExchangeAnalyzer
+
+#This function checks the pagefile min and max sizes
+Function Run-WSSRV004()
+{
+
+   [CmdletBinding()]
+    param()
+
+    $TestID = "WSSRV004"
+    Write-Verbose "----- Starting test $TestID"
+
+    $PassedList = @()
+    $FailedList = @()
+    $WarningList = @()
+    $InfoList = @()
+    $ErrorList = @()
+
+    foreach ($server in $exchangeservers) {
+        $name = $server.name
+
+        # Get RAM of remote server
+        $RAMinMB = (Get-WmiObject -Class win32_physicalmemory -computer $name | measure-object -property capacity -sum).sum/1048576
```

Comment:
I would test that the WMI object you get back is valid before piping it to measure-object. There's so many reasons WMI can fail so I would plan on it.

Also, can you define a constant for this 1048576 value and divide by that?